chipy_org/templates/envelope/contact.html

```diff
@@ -10,7 +10,9 @@ <h1>{% trans 'Contact us' %}</h1>
         <div id="contact">
           <form method="post" id="contact-us" action=".">
             {% csrf_token %}
-            {% antispam_fields %}
+            {% autoescape off %}
+                {% antispam_fields %}
+            {% endautoescape %}
```

Comment:
We have CAPTCHA already, do we need this too?  I would like to remove the envelope library if possible.  Need to see if this provides anything beyond a standard form/view/route.